src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+@import "./blocks/variables.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $day-padding;
+  width: 756px;
+  gap: $gap;
+  margin: auto;
```

Comment:
Calendar should be in the middle of the screen. Margin: auto work only horizontally :)